synapse/cortex.py

```diff
@@ -3131,15 +3134,13 @@ async def loadCoreModule(self, ctor, conf=None):
 
         return modu
 
-    async def _loadCoreMods(self, ctors):
+    async def _loadCoreMods(self):
 
         mods = []
-
         cmds = []
         mdefs = []
 
-        for ctor in ctors:
-
+        async def loadit(ctor, custom=False):
```

Comment:
any particular reason for using an inner function here rather than an underscore method?

Reply:
nope - that was just for hacking around the concept so it was easy to load modules with the additional context that it wasn't a default "core" module that we controlled.

Reply:
For things that we mark as deprecated we know it and shouldn't need to warn on it for loading; but for external users we definitely need to warn that they are using something which we've marked as deprecated.

Reply:
Yea, i think we just wanna warn once per snap though.  If they happen to run a query that creates a million, we dont wanna inundate them with depr warns

Reply:
We just needed to separate the knowledge of "our internal models" vs "anything else being loaded via a ctor". this has been streamlined but all our little module loading funcs also need some one line docstrings to note what they do for future selves.